### DISCUSSION TITLE
Single-end vs paired-end bulk RNA-seq data

### DISCUSSION THREAD

**Discussion by JoostKoedijk** [milaboratory/mixcr#1838] [Q&A] [answered]
Hi all, 

For my MiXCR analyses applied to bulk RNA-sequencing data, I have worked with single-end data to generate the clonotype data. What is your experience on using single-end vs bulk RNA-seq data? Would single-end be enough or is it really a limitation if I only use single instead of paired? 

Many thanks again!

Best wishes,
Joost

**Comment by mizraelson** [comment, marked as answer]
Hi Joost,
It depends on the size of the fragments in your library. If your library fragments are the same length as the single-end read length, it shouldn’t make a significant difference since the second read would have covered the same sequence anyway. However, if your library fragments are longer, you might lose some reads that could otherwise cover the CDR3 region if the R2 was present. Generally speaking, bulk RNA-seq (whether single or paired-end) yields fewer clones compared to targeted approaches.
